app/controllers/api/base_controller.rb

```diff
@@ -3,6 +3,8 @@
 class Api::BaseController < ApplicationController
   skip_before_action :verify_authenticity_token
 
+  before_action :require_project
```

Comment:
I'm just mimic-ing the concern we have. Require everywhere it's included and skip when needed.